### PR TITLE
chore(ci): use release mode to run integration

### PR DESCRIPTION
**Pull request by doitian**

### CODE REVIEW

**Comment by doitian** [comment]
@nervos-bot-user try integration

**Comment by u2** [comment]
Because storing the cache is very slow? CITA is release mode also.

**Comment by doitian** [comment]
> Because storing the cache is very slow? CITA is release mode also.

Debug mode can share the cache with test. That why I also added a environment variable on the job so the integration test job now will use their own cache.

**Comment by doitian** [comment]
@nervos-bot-user try integration

**Comment by doitian** [comment]
@nervos-bot-user try integration

**Comment by doitian** [comment]
@nervos-bot-user try integration

**Comment by doitian** [comment]
@nervos-bot-user try integration

File: Makefile [hunk 1]
```diff
@@ -25,16 +25,17 @@ setup-ckb-tools:
...
 integration: setup-ckb-test ## Run integration tests in "test" dir.
 	cargo build ${VERBOSE}
-	cd test && cargo run ../target/debug/ckb
+	cd test && cargo run ${VERBOSE} -- ../target/debug/ckb
 
 integration-windows:
 	cp -f Cargo.lock test/Cargo.lock
 	cargo build ${VERBOSE}
-	cd test && cargo run ../target/debug/ckb
+	mv target test/
```
Comment (keroro520) [inline Makefile]: Why `mv` here?

Comment (doitian) [reply Makefile]: ln is not available in Windows

**Comment by doitian** [comment]
I have rolled back the Windows changes, which still run debug mode integration now. In the future, we may try to switch to release mode to see whether it is faster.

**Pull request merged**